src/editor/Collaboration.js

```diff
@@ -10,7 +10,9 @@ define(function (require, exports, module) {
             // the id/element dom element that will hold remote videos
             remoteVideosEl: 'remotesVideos',
             // immediately ask for camera access
-            autoRequestMedia: false
+            autoRequestMedia: false,
+            // TODO : Shift this to config.
+            url: "localhost:8888"
```

Comment:
Can you file an issue for this in Thimble? Ideally, we want this set via the bramble API on initialization and we can fallback on localhost.